### PR TITLE
Uppercase first languages letters

### PR DESCRIPTION
**Pull request by AHOHNMYC**
Because some languages started from uppercase and others from lower case.

### CODE REVIEW

**Review by wxiaoguang — APPROVED** [review]
Yup, that's also a longstanding question in my mind why some of the first letters are lower-cased. 😊

I think it's correct to make them upper-cased.

**Comment by delvh** [comment]
I asked the same question once in #18211:
> Now, it is also visible that for some reason many locales have lowercase names while others have uppercase names.
Should we unify them to all be uppercased or lowercased (where possible)?

With the corresponding answer from @zeripath:
> The problem is that in most romance languages the language does not have an upper case letter.
However, I think it's valid in almost all languages that the first letter of a list should be capitalized if any of the others are capitalized so I think we can validly force ToTitle when rendering the languages as a list like this.

I'm not entirely sure if that's now a "yes please" or a "please no"...

**Comment by AHOHNMYC** [comment]
@wxiaoguang yess! Absolutely in every first project French and Russian languages starts from lower case. This is the greatest mystery in FOSS world :)

**Comment by lunny** [comment]
> I did a quick google search (and wikipedia search), most language names can have the first letter upper-cased.
> 
> The only exceptions seem to be `Српски` and `Ελληνικά` (if I understand correctly ...)

But then they are not native. Just like you want an Uppercase of Chinese.

**Review by delvh — APPROVED** [review]
Generally LGTM, if no one has any objections about the upper case names of the languages.
Do we know by now how to display `српски` and `ελληνικά`?

File: custom/conf/app.example.ini
Comment (wxiaoguang) [inline custom/conf/app.example.ini:2128]:
```suggestion
;NAMES = English,简体中文,繁體中文（香港）,繁體中文（台灣）,Deutsch,Français,Nederlands,Latviešu,Русский,Українська,日本語,Español,Português do Brasil,Português de Portugal,Polski,Български,Italiano,Suomi,Türkçe,Čeština,српски,Svenska,한국어,ελληνικά,فارسی,Magyar nyelv,Bahasa Indonesia,മലയാളം
```

File: docs/content/doc/advanced/config-cheat-sheet.en-us.md
Comment (wxiaoguang) [inline docs/content/doc/advanced/config-cheat-sheet.en-us.md:1003]:
```suggestion
- `NAMES`: **English,简体中文,繁體中文（香港）,繁體中文（台灣）,Deutsch,Français,Nederlands,Latviešu,Русский,Українська,日本語,Español,Português do Brasil,Português de Portugal,Polski,Български,Italiano,Suomi,Türkçe,Čeština,српски,Svenska,한국어,ελληνικά,فارسی,Magyar nyelv,Bahasa Indonesia,മലയാളം**: Visible names corresponding to the locales
```

File: modules/setting/i18n.go
Comment (wxiaoguang) [inline modules/setting/i18n.go:29]:
```suggestion
	"sr-SP", "српски",
```

File: modules/setting/i18n.go
Comment (wxiaoguang) [inline modules/setting/i18n.go:32]:
```suggestion
	"el-GR", "ελληνικά",
```

**Review by wxiaoguang — COMMENTED** [review]
~~I would like to revert to `српски` and `ελληνικά`.~~

~~Let's keep this PR open for a few days before merge.~~

----

Update: it seems that no problem for these two, I search again, the result is as below.

So, let's trust Google and Wikipedia and keep them.

image

image

**Review by 6543 — APPROVED** [review]

**Pull request merged**
